src/js/src/components/SearchFacetOptions.vue

```diff
@@ -77,17 +77,30 @@ export default {
       let appliedFacets = this.searchAppliedFacets.join('')
       this.addSpecificRequestedFacets({facet:facetArea, query:structuredQuery, appliedFacets:appliedFacets})
     },
-    applyFacet(facetCategory, facet) {
+    applyFacet(facetCategory, facet, event) {
       let facetAllreadyApplied = false
       let newFacet = '&fq=' + facetCategory + ':"' + facet + '"'
       this.searchAppliedFacets.forEach((facet) => {
         facet === newFacet ? facetAllreadyApplied = true : null
       })
       if(!facetAllreadyApplied) {
-        this.updateSolrSettingOffset(0)
-        this.addToSearchAppliedFacets(newFacet)
-        this.$_pushSearchHistory('Search', this.query, this.searchAppliedFacets, this.solrSettings)
-    }
+          if(event.ctrlKey === true) {
```

Comment:
Good to make use of the 'event' to handle the ctrlKey issue but you need to handle 'command' to include Mac-user too. So I propose the following correction to hande the Apple Command key:

`if(event.ctrlKey || event.metaKey) {` // which also takes my comment below into account:) 

Instead of saying anything about the evaluation of 'true' you have here I will just use three of these ...

Reply:
I must admit i totally forgot about mac keyboards. Damn, good catch. Fixed that now.